hadoop-hdds/common/src/main/java/org/apache/hadoop/ozone/OzoneConfigKeys.java

```diff
@@ -355,10 +355,6 @@ public final class OzoneConfigKeys {
   public static final String OZONE_HTTP_FILTER_INITIALIZERS_KEY =
       "ozone.http.filter.initializers";
 
-  public static final String OZONE_CONTAINER_COPY_WORKDIR =
```

Comment:
Please add this key to OzoneConfiguration#addDeprecatedKeys.

Reply:
Updated, please have a look.